src/README.md

```diff
@@ -77,18 +82,31 @@ myApp.state.sfxVolume.subscribe(result => console.log('Value Between 0-1', resul
 myApp.state.pointerSize.subscribe(result => console.log('Value Between 0-1', result));
 myApp.state.buttonSize.subscribe(result => console.log('Value Between 0-1', result));
 myApp.state.controlSensitivity.subscribe(result => console.log('Value Between 0-1', result));
-myApp.state.removableLayers.subscribe(result => console.log('Object containing the layers and their state(boolean)', result));
-myApp.state.hudPosition.subscribe(result => console.log('String position of the HUD', result));
+myApp.state.removableLayers.subscribe(result => console.log('Value Between 0-1', result));
+myApp.state.hudPosition.subscribe(result => console.log('String position of the HUD', result));*
 ```
 
+*the HUD position state requires one additional bit of configuration to interact with Springroll Container correctly. The example below shows the `respond` call you need to implement to report the HUD positions your game allows.
+
+```javascript
+var myApp = new springroll.Application({
+  features: {
+    hudPosition: true
+  }
+});
+
+myApp.container.respond('hudPositions', ['top', 'bottom']); //this should be an array of strings(representing the positions the game supports)
```

Comment:
For this example, I would instead provide a list of ALL possible options (which I assume to be `['top', 'bottom', 'left', 'right']`). I would _then_ indicate with a comment that any subset can be used if necessary.